test/infra/infra-dbdeployer-mariadb10-galera/docker/entrypoint.sh

```
@@ -0,0 +1,153 @@
+MYSQL_CMD="mysql -h127.0.0.1 -uroot -p${DBDEPLOYER_ROOT_PASS}"
+NODE_PORTS=(3306 3307 3308)
+NODE_NAMES=("node1" "node2" "node3")
+
+# Wait for all 3 nodes to accept connections
+for i in 0 1 2; do
+    PORT="${NODE_PORTS[$i]}"
+    NAME="${NODE_NAMES[$i]}"
+    echo -n "Waiting for ${NAME} on port ${PORT}..."
+    MAX_WAIT=120
+    COUNT=0
+    while ! ${MYSQL_CMD} -P${PORT} -e "SELECT 1" >/dev/null 2>&1; do
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)